src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveforbarn/GjeldendeBarnRepository.kt

```diff
@@ -17,7 +17,7 @@ interface GjeldendeBarnRepository :
     @Query(
         """
         SELECT b.id behandling_id, pi.ident fodselsnummer_soker, bb.person_ident fodselsnummer_barn, 
-          bb.fodsel_termindato termindato_barn, FALSE AS fra_migrering
```

Comment:
Er det avklart med fag at vi dropper de som bare har termindato?

Reply:
Sjekket ikke med fag, men det burde jeg sikkert gjøre. Jeg fjernet det fordi jeg tok noen stikkprøver, og så at aktuelle barn som skulle lages oppgave hadde fødselsnummer, også tenkte jeg at det er vel noe galt dersom et barn ikke har fått fødselsnummer 6 mnd etter fødsel?

Reply:
Konkluderer med at det sikkert er lurt å hente fnr fra pdl for terminbarn, da de burde ha fått fødselsnummer innen 6 mnd etter fødsel etter denne diskusjonen: https://nav-it.slack.com/archives/C01087QRJ2U/p1661778825749399
Enklere for feilsøking og ryddigere i db dersom alle har fødselsnummer. (Om de ikke har fødselsnummer i PDL etter 6 mnd har det skjedd noe annet galt, men det kan i såfall bare logges)